app/helpers/storage_resource_helper/textual_summary.rb

```diff
@@ -48,8 +48,8 @@ def textual_physical_storage
   end
 
   def textual_group_capabilities
-    capabilities = @record.capabilities.map do |c|
-      [c["name"], c["value"]]
+    @record.capabilities.flat_map do |key, values|
```

Comment:
should the `capabilities` be assigned here so that it can be used in line 58?

Reply:
fixed